api/models/tearun.ts

```diff
@@ -1,11 +1,12 @@
+import { GasificationPower } from './gasification-power';
 import { GenericPowerOnly } from './generic-power-only';
 import { InputVarMod } from './tea.model';
 
 function calculate(input: InputVarMod) {
 
-    const result = GenericPowerOnly(input);
-
-    return result;
+    const ResultFromGasificationPower = GasificationPower(input);
+    const ResultFromGenericPowerOnly = GenericPowerOnly(input);
+    return { ResultFromGasificationPower, ResultFromGenericPowerOnly };
```

Comment:
You may change it after dividing input. You could add "model" as one of input and use switch to handle different models. For example, {"model": generic-power-only} if input is for GenericPowerOnly.